### PR TITLE
Harden NEG GC

### PR DESCRIPTION
**Pull request by freehan**
Modified 2 things:
- make NEG GC more frequent
- delay initial NEG GC to allow services to be fully processed.

### CODE REVIEW

File: pkg/neg/controller.go [hunk 1]
```diff
@@ -41,6 +41,11 @@ import (
 	"k8s.io/ingress-gce/pkg/utils"
 )
 
+const (
+	// gcPeriod is the interval between NEG garbage collection workflows
+	gcPeriod = 2 * time.Minute
```
Comment (bowei) [inline pkg/neg/controller.go]: make this a flag

File: pkg/flags/flags.go [hunk 2]
```diff
@@ -215,6 +216,8 @@ L7 load balancing. CSV values accepted. Example: -node-port-ranges=80,8080,400-5
 		`This flag is deprecated. Use -v to control verbosity.`)
 	flag.Bool("use-real-cloud", false,
 		`This flag has been deprecated and no longer has any effect.`)
+	flag.DurationVar(&F.GCPeriod, "gc-period", 120*time.Second,
+		`Relist and garbage collect cloud resources this often.`)
```
Comment (bowei) [inline pkg/flags/flags.go]: isn't this just for NEG? (the description is a little misleading...)

**Comment by freehan** [comment]
ping

**Pull request merged**